docs/update_and_migration/from_3.3/to_4.0.md

```diff
@@ -159,6 +159,12 @@ If the dry run is successful, run:
 php bin/console ibexa:migrations:migrate
 ```
 
+### Migrate richtext namespaces
+
+Run the upgrade script for updating xml namespaces inside richtext fields:
+
+`./bin/console ibexa:migrate:richtext-namespaces`
```

Comment:
This is inconsistent with what I can see in the lines 158-160 above. I'm not able to add suggestion because it includes code block itself. This block should be multiline, specify bash and call php CLI directly.

Reply:
FYI : I have fixed that in bb617449991a5405c81a63926aab41747470262d and 397fb6c76af7ba15c3eb3a45d996cfaab90ff457